Source/Charts/Components/YAxis.swift

```diff
@@ -150,6 +150,17 @@ open class YAxis: AxisBase
         var min = _customAxisMin ? _axisMinimum : dataMin
         var max = _customAxisMax ? _axisMaximum : dataMax
         
+        // Make sure max is greater than min
+        if min > max {
```

Comment:
will take a look if this is the right place to fit. I wonder why there is min > max scenario, it should be solved earlier. but thanks for finding the cause.

Reply:
``` swift
var min = _customAxisMin ? _axisMinimum : dataMin
var max = _customAxisMax ? _axisMaximum : dataMax
```
These lines of code caused min>max
Look example below：
```swift
chartView.leftAxis.axisMaximum = -40.0
let yVals = [
            BarChartDataEntry(x: 0, y: -10),
            BarChartDataEntry(x: 1, y: -20),
            BarChartDataEntry(x: 2, y: -30),
            BarChartDataEntry(x: 3, y: -15)
]
.....
var min = dataMin // dataMin = -30
var max = _axisMaxium // Manually set to 40
```